### PR TITLE
Docs update 2

### PR DESCRIPTION
**Pull request by zehawki**

### CODE REVIEW

File: docs/editor.md [hunk 1]
```diff
@@ -29,41 +32,50 @@ const App = () => {
...
+### Editing
+[Link to editing example](examples/pages/examples/simple.tsx)
+
+[Live demo of the editing example](//demo/simple)
+
+### Viewing
+[Link to viewing example](examples/pages/examples/readonly.tsx)
+
+[Live demo of the viewing example](//demo/readonly)
+
```
Comment (macrozone) [inline docs/editor.md:44]: nice

File: docs/editor.md [hunk 2]
```diff
@@ -29,41 +32,50 @@ const App = () => {
...
 ## Props
 
 ### `value: Value`
 
-this is the value that the editor should display. You will usually load that from a database.
-It has the type `Value`, but should generally considered to be "opaque", so don't rely on its internal structure.
+This is the content that the editor will display. The data for this prop may come from any source, eg a file or database or API etc. It is a JSON representation of the entire content that should be rendered. This information is "opaque", and is not meant to be edited directly under normal circumstances, so don't rely on its internal structure.
+
```
Comment (macrozone) [inline docs/editor.md:50]: good

**Comment by macrozone** [comment]
@zehawki i wanted to give feedback, but then i noticed, that some changes are already applied. 

Can you please rebase it first and resolve conflicts?

**Comment by zehawki** [comment]
@macrozone can you please chekc now... I'm not very sure Ive done it correctly. However, when I check under commits and click on the commit link (https://github.com/react-page/react-page/pull/931/commits/104b06856c313dd050d6f81102661ebab25db3bb), I can see only my fresh changes. Thanks.

**Comment by macrozone** [comment]
> @macrozone can you please chekc now... I'm not very sure Ive done it correctly. However, when I check under commits and click on the commit link ([104b068](https://github.com/react-page/react-page/commit/104b06856c313dd050d6f81102661ebab25db3bb)), I can see only my fresh changes. Thanks.

still conflicting, i'll take over

**Comment by zehawki** [comment]
@macrozone Please see now.

**Comment by macrozone** [comment]
awesoem, thx!

**Comment by zehawki** [comment]
@macrozone could you please check why the image is not showing up here: https://react-page.github.io/docs/#/slate?id=slate-as-the-rte

image


Here's how its supposed to be: https://github.com/react-page/react-page/blob/master/docs/slate.md (with the animated gif)

**Comment by zehawki** [comment]
So 3 sections are done - "getting started", "editor component" and "rich text editing". Will work on "builtin cell plugins" today and next "custom cell plugins"

**Pull request merged**